src/main/resources/application.properties

```diff
@@ -5,4 +5,4 @@ spring.datasource.password=
 spring.jpa.database-platform=org.hibernate.dialect.H2Dialect
 spring.jpa.hibernate.ddl-auto=update
 spring.h2.console.enabled=true
-
+spring.jpa.properties.hibernate.jdbc.time_zone=UTC
```

Comment:
I don't think we need this property. At least it didn;t do anything for me.

Reply:
It set DB fields with data to has UTC time. It was one of things you were asking for before, wasn't ?

Reply:
Yeah, but i dont think this property is actually doing it. Or is it? Like I've said, I set db timezone to UTC in db connection url not with this property because it didn't work for me.